packages/dapi-grpc/src/lib.rs

```diff
@@ -13,3 +13,33 @@ pub mod platform {
         include!("platform/proto/org.dash.platform.dapi.v0.rs");
     }
 }
+
+#[cfg(test)]
+mod test {
+    use crate::platform::v0::platform_client::PlatformClient;
+
+    #[cfg(feature = "testnet")]
+    #[tokio::test]
+    async fn testnet_request() {
```

Comment:
1. functional tests should be in test folder
2. testing against testnet is not reliable because testnet could contain incompatible version or just platform might be down